tests/easier68k/simulator/test_m68k.py

```diff
@@ -167,6 +167,17 @@ def test_full_integration():
 
     list_file = ListFile()
 
+    '''
+    ORG 1036
+MAGIC:
+    DC.W $ABCD
+    ORG    1024
+START:
+    MOVE.W #$ABCD, ($00AAAAAA).L
+    SIMHALTe
```

Comment:
The trailing 'e'?

Reply:
It's from a more elegant language, hahahaha